src/channel_internal.c

```diff
@@ -85,11 +94,11 @@ void channel_internal_close(THANDLE(CHANNEL_INTERNAL) channel_internal)
     srw_lock_release_exclusive(channel_internal_ptr->lock);
 }
 
-IMPLEMENT_MOCKABLE_FUNCTION(, THANDLE(CHANNEL_INTERNAL), channel_internal_create_and_open, THANDLE(THREADPOOL), threadpool)
+IMPLEMENT_MOCKABLE_FUNCTION(, THANDLE(CHANNEL_INTERNAL), channel_internal_create_and_open, THANDLE(PTR(LOG_CONTEXT_HANDLE)), log_context, THANDLE(THREADPOOL), threadpool)
```

Comment:
>threadpool [](http://example.com/codeflow?start=159&length=10)

is `threadpool` == `NULL` valid argument?

Reply:
Validity of arguments is checked by channel.c